setup.py

```diff
@@ -5,13 +5,14 @@
 
 from setuptools import setup, find_packages
 
+from cli import __version__
```

Comment:
can this be local import?

Reply:
I don't think so, as `setup.py` is outside of the `cli` module.